components/dashboard/src/onboarding/StepUserInfo.tsx

```diff
@@ -22,7 +22,8 @@ export const StepUserInfo: FC<Props> = ({ user, onComplete }) => {
 
     const [firstName, setFirstName] = useState(first);
     const [lastName, setLastName] = useState(last);
-    const [emailAddress, setEmailAddress] = useState(User.getPrimaryEmail(user) ?? "");
+    // Email purposefully not pre-filled
+    const [emailAddress, setEmailAddress] = useState("");
```

Comment:
This has the risk that if an Update is trigerred, it actually sets the email to the empty value. Getting a preview to validate this.

Reply:
Looks the client-side validation does prevent a class of whitespace email so it should be okay here